google/cloud/bigtable/testing/mock_response_reader.h

```diff
@@ -111,6 +113,15 @@ class MockAsyncResponseReader
   MOCK_METHOD(void, StartCall, (), (override));
   MOCK_METHOD(void, ReadInitialMetadata, (void*), (override));
   MOCK_METHOD(void, Finish, (Response*, grpc::Status*, void*), (override));
+
+  // Preserve the behavior from before https://github.com/grpc/proposal/pull/238
+  // of not destroying the object when
+  // std::unique_ptr<grpc::ClientAsyncResponseReaderInterface> goes out of
+  // scope.
+  //
+  // TODO(jacobsa): mark this with the override keyword once the method exists
```

Comment:
nit: we do not create `TODO(person)`, I created #6566 for your, please change this to `// TODO(#6566)`

Reply:
I was wondering about that. Done, thanks.